src/_data/members/bryanhealey.json

```diff
@@ -0,0 +1 @@
+{"username": "bryanhealey"}
```

Comment:
This may have not been totally clear in the instructions, but the username here should be your GitHub username - in your case `healeyb`. Swapping that out should get this working!